### PR TITLE
Update The Theme Text Dynamically (issue #193)

### LINKED ISSUE #193: Update Theme Text Display in MorePage.tsx when Switching Themes
Description:
Currently, when switching between dark mode and light mode in MorePage.tsx, the text indicating the current mode does not update accordingly, leading to confusion for the user. To improve clarity and user experience, update the state of the text to reflect the current theme.
When in dark mode, the text should display "Dark Mode" in white.
When in light mode, the text should display "Light Mode" in black.

Tasks:

- [x]  Review the current theme switch functionality in MorePage.tsx.
- [x]  Update the code to change the text based on the selected theme:
- [x] "Dark Mode" should display in white text when dark mode is active.
- [x] "Light Mode" should display in black text when light mode is active.
- [x]  Ensure that the theme text updates dynamically when the user switches between themes.
- [x]  Write test cases to validate the text change when switching between themes.
- [x]  Test on both iOS and Android devices to confirm the behavior works as expected.

Acceptance Criteria:
 

- [x] The text indicating the current theme should update correctly:
    - [x] White text for "Dark Mode."
    - [x] Black text for "Light Mode."
- [ ]  The changes should not affect other functionalities on the MorePage.tsx.
- [ ]  The theme switch should work seamlessly across devices.
- [ ]  All test cases must pass, and the app should run smoothly without any issues after the modification.

reference

Don't change the alignment like below, update the text indicating current mode like below.

### PR DESCRIPTION
**Pull request by teamomiamigo**
**Fixes #193:** Update Theme Text to Reflect Dark/Light Mode Changes in MorePage
This PR updates the theme text in MorePage.tsx to reflect the current mode, whether dark or light. The text now clearly indicates "Dark Mode" or "Light Mode" and displays in the appropriate color, enhancing user clarity when switching between themes.

**Why Was It Changed?**
Previously, when switching between dark mode and light mode, the text indicating the current theme did not update accordingly. By ensuring that the theme text changes dynamically, users can easily identify the current mode they are in, improving the overall user experience.

**How Was It Changed?**
The theme switch functionality in MorePage.tsx was updated to modify the text based on the selected theme:
"Dark Mode" is displayed in white text when dark mode is active.
"Light Mode" is displayed in black text when light mode is active.
Additional test cases were created to validate the text changes when switching between themes. 

**UI After Changes**
After the changes, when the user toggles between dark mode and light mode, the theme text updates dynamically. When in dark mode, it clearly displays "Dark Mode" in white, and when in light mode, it displays "Light Mode" in black.

### CODE REVIEW

File: lib/screens/MorePage.tsx [hunk 1]
```diff
@@ -145,7 +145,7 @@ export default function MorePage() {
           <View style={styles.textContainer}>
             <View style={styles.buttonContainer}>
               <View style={[styles.switchContainer, { backgroundColor: theme.background }]}>
-                <Text style={[styles.switchText, { color: theme.text }]}>Dark Mode</Text>
+                <Text style={[styles.switchText, isDarkmode ? styles.darkModeText : styles.lightModeText]}>{isDarkmode ? "Dark Mode" : "Light Mode"}</Text>
```
Comment (AmarHadzic) [inline lib/screens/MorePage.tsx:148]: I would suggest adding some comments on your code.

File: lib/screens/MorePage.tsx [hunk 2]
```diff
@@ -285,6 +285,12 @@ const styles = StyleSheet.create({
     height: undefined,
     resizeMode: "cover",
   },
+  lightModeText: {
```
Comment (AmarHadzic) [inline lib/screens/MorePage.tsx:288]: Nice and simple code here. Very understandable without the need to use comments.

File: __tests__/MoreScreen.test.tsx [hunk 3]
```diff
@@ -106,14 +105,70 @@ describe('MorePage', () => {
       "mailto:yashkamal.bhatia@slu.edu?subject=Bug%20Report%20on%20'Where's%20Religion%3F'&body=Please%20provide%20details%20of%20your%20issue%20you%20are%20facing%20here."
     );
   });
-  it('renders the "Logout" button', () => {
-    const { getByText } = render(
+
```
Comment (AmarHadzic) [inline __tests__/MoreScreen.test.tsx:108]: Good test cases and good comments. However some comments can be better, like comment on what it is testing and what the value should be.

**Review by AmarHadzic — COMMENTED** [review]
Overall, good job!

**Review by rcAsironman — CHANGES_REQUESTED** [review]
@teamomiamigo The code is working as expected, but please use the theme object to apply the themes instead of creating separate CSS properties on line 148.

**Comment by teamomiamigo** [comment]
attached are the changes made on line 148 of MorePage that uses the theme object to ensure the color of the text.

**Review by rcAsironman — CHANGES_REQUESTED** [review]
@teamomiamigo All checks have failed. Please review your work before submitting, and run it on your local machine first.

**Review by rcAsironman — APPROVED** [review]

**Review by rcAsironman — APPROVED** [review]

**Pull request merged**